prompts/index.js

```diff
@@ -206,73 +209,71 @@ const continuePrompts = [
     name: 'continue',
     message: `\nPress enter to continue →`,
     afterMessage: ' ',
-    transformer: input => ' ',
   },
 ]
 
-const searchIncidentsPrompt = [
-  {
-    type: 'datepicker',
-    name: 'since',
-    message: `What is the start of the date range over which you want to search? ${reset.green(
-      '→'
-    )}`,
-    format: ['mm', '/', 'dd', '/', 'yy', ' ', 'hh', ':', 'MM', ' ', 'TT'],
-  },
-  {
-    type: 'datepicker',
+async function getSearchQuery() {
+  // Datetime prompt does not currently support formatting of the date @ sdk v2.0.1
```

Comment:
Create an issue in the 'support' repo in the ops group about this @tw5033 if it's a feature we want!

Reply:
done!